mystyle.css

```diff
@@ -0,0 +1,299 @@
+body {
+  background-color: #0d0d15;
+  font-family: "poppins", Aral, sans-serif;
+  font-size: 18px;
+  line-height: 28px;
+  color: #f5f8fb;
+}
+
+header {
+  background-color: #0d0d15;
+  width: 1728px;
+  height: 112px;
+}
+
+.binabox-logo img{
+  background-color: #0d0d15;
+  width: 36px;
+  height: 40px;
+}
+
+ul,
+ol {
+  list-style: none;
+}
+
+a {
+  text-decoration: none;
+}
+
+.button {
+  background-color: #0265dc;
+  font-family: "poppins", Aral, sans-serif;
+  font-size: 18px;
+  font-weight: 600;
+  line-height: 28px;
+  color: #f5f8fb;
+  border: 0px solid #0265dc;
+  width: 159px;
+  height: 48px;
+}
+
+.button-is-primary {
+  background: transparent;
+  font-family: "poppins", Aral, sans-serif;
+  font-size: 18px;
+  font-weight: 600;
+  line-height: 28px;
+  color: #f5f8fb;
+  border: 1px solid rgb(208, 215, 251, 17%);
+  width: 150px;
+  height: 49px;
+}
+
+.top-nav {
+  display: flex;
+  justify-content: space-between;
+}
+
+.nav-list {
+  display: flex;
+  column-gap: 20px;
+}
+
+.nav-buttons {
+  display: flex;
+  align-items: center;
+  column-gap: 16px;
+}
+
+.nav-buttons::before {
+  content: "";
+  width: 6px;
+  height: 20px;
+  background: red;
+}
+
+.search-form {
+  order: -1;
+}
+
+.home {
+  text-align: center;
+}
+
+.home p {
+  font-family: "poppins", Aral, sans-serif;
```

Comment:
`font-family` is an inherited property, it does not need to be specified for each selector